packages/core/src/percy.js

```diff
@@ -321,6 +325,18 @@ export class Percy {
     }.call(this));
   }
 
+  // automateScreenshot
+  async automateScreenshot(options) {
```

Comment:
core package shouldn't not make call to selenium server it should be present in selenium-appium-utils (cli-poa currently) package . the screenshots and other stuff should be posted to local SDK server

Reply:
core package takes all information from SDKs and posts it to `@percy/client` package it would act as a mediator only

Reply:
I understand the part that selenium server calls should be made from `selenium-appium-utils` The screenshot and other stuff should be posted to local SDK server @itsjwala  I didn't get that part. We had discussed that the calls for screenshot and other stuff would go via CLI only. if you meant that we need to ship to some different folder in cli that's fine.

Reply:
discussed with Neel 

move to `webdriver-utils`  (decide name change for `cli-poa`) and import single function in api.js that will be called.

Reply:
@itsjwala  / @Amit3200 can you please add `emote` and confirm whether we are good to mind this review comment as `Resolved` ?